data/Guyana/National_Assembly/ep-popolo-v1.0.json

```diff
@@ -22,6 +22,17 @@
       "name": "Rupert Roopnarine"
     },
     {
+      "honorific_prefix": "Dr.",
+      "id": "133bdcf4-8df1-4207-be10-1a0ff8e6c0af",
+      "image": "/assets/images/no-image-person.jpg",
```

Comment:
There are quite a few URL here that are relative rather than absolute (though ones like this look like they should probably be skipped anyway…)